src/main/java/ch/securify/decompiler/ControlFlowDetector.java

```diff
@@ -221,6 +221,12 @@ else if (opcode == OpCodes.RETURN) {
 				richBranches.put(pc, BRANCH_EXIT);
 				return;
 			}
+			else if (opcode == OpCodes.REVERT) {
```

Comment:
As REVERT opcode is added in ControlFlowDetector. Do we need to add StaticCall also here, in this class?

Reply:
This is computing the internal control flow graph. So I don't think it is needed here. But we still need to tag the output of StaticCall as uncontrolled. For this I need to synch with @ptsankov on the correct memory abstraction.